.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - '**'
+    tags-ignore:
+      - '**'
+    paths-ignore:
+      - '**/CHANGELOG.md'
+      - '**/package.json'
+  pull_request:
+  workflow_dispatch:
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set Node Version
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - name: Install dependencies
+        run: npm i
+
+      - name: Bootstrap
+        run: npm run bootstrap
+
+      - name: Generate docs
+        run: npm run build:docs
+
+      - name: Semantic Release
+        uses: cycjimmy/semantic-release-action@v2
+        env:
```

Comment:
I think you might need the `SEMANTIC_RELEASE_PACKAGE` env var here too. See here: https://github.com/hirosystems/gh-actions-example/blob/main/.github/workflows/ci.yml#L134

Reply:
Added `SEMANTIC_RELEASE_PACKAGE ` env variable as mentioned.